libs/ui-toolkit/modals/src/lib/ui-modal/ui-loading-modal/ui-loading-modal.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
+import { UiLoadingModalComponent, UiSimpleModalData, UiModalModule } from '../../..';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { Component, Inject, Input, OnChanges, OnInit } from '@angular/core';
+import { MatDialogRef, MatDialog, MAT_DIALOG_DATA, MatDialogModule } from '@angular/material';
+import { PipesModule } from '@gorilainvest/ui-toolkit/pipes';
+
+@Component({
+  selector: 'gor-test',
+  template: `
+    gor-test
+  `,
+  styles: ['./ui-simple-modal.component.scss']
+})
+export class TesteComponent implements OnChanges, OnInit {
+  constructor(
+    public dialogRef: MatDialogRef<TesteComponent>,
+    public dialog: MatDialog,
+    @Inject(MAT_DIALOG_DATA) public data: UiSimpleModalData
```

Comment:
Check properties' access modifiers and only use `public` if you need to access them in the template or use them as input